src/lxc/tools/lxc_attach.c

```diff
@@ -371,7 +371,7 @@ static int get_pty_on_host(struct lxc_container *c, struct wrapargs *wrap, int *
 	lxc_mainloop_close(&descr);
 err2:
 	if (ts && ts->sigfd != -1)
-		lxc_console_sigwinch_fini(ts);
+		lxc_console_signal_fini(ts);
```

Comment:
But...  didn't you rename this in the previous patch?  This is non-bisectable right?

Reply:
Right, squashing.